.github/workflows/release.yml

```diff
@@ -180,7 +180,7 @@ jobs:
         working-directory: fastlane
```

Comment:
@mykdavies this is what you can copy for your testlfight build workflow. copy starting at line 179 and make a new workflow for Release to test flight.

https://litoarias.medium.com/continuous-delivery-for-ios-using-fastlane-and-github-actions-edf62ee68ecc

This is a good resource then you can add the secrets in the repo settings page and reference the secret variable in this .yml file.